langs/julia.yaml

```diff
@@ -4,12 +4,12 @@ aliases:
 name: "Julia"
 
 install:
-  prepare:
-    apt:
-      - julia
-  apt:
-    - julia
   manual: |
+    wget https://julialang-s3.julialang.org/bin/linux/x64/1.7/julia-1.7.0-linux-x86_64.tar.gz
+    tar xvf julia-*.tar.gz
+    rm julia-*.tar.gz
```

Comment:
We don't actually need to clean things up since this script is run in a temporary directory.